pkg/attestation/crafter/api/attestation/v1/crafting_state.proto

```diff
@@ -146,10 +147,21 @@ message PolicyEvaluation {
   // Evaluation messages, intended to communicate evaluation errors (invalid input)
   repeated string skip_reasons = 14;
 
+  // Group this evaluated policy belongs to, if any
+  Reference policy_reference = 15;
```

Comment:
where are you setting the policy_reference value? I can't see any changes in the crafter pkg?

Reply:
This is set during policy evaluation, which happens in the crafting stage, see `policies.go` below.

Reply:
oh I see, thanks